src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -74,8 +74,10 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
   hotRegisterer: HotTableRegisterer = new HotTableRegisterer();
   newStudentsHOT: string = 'newStudentsHOT';
 
-  enrollData?: string;
+  // enum
+  panelClass: typeof PanelClass = PanelClass;
```

Comment:
BTW, for enum it should be defined on the top of the class. The name should be `PanelClass` since it is more like a constant.